content/chronograf/v1.8/administration/managing-security.md

```diff
@@ -417,7 +421,9 @@ JWKS_URL="https://example.com/adfs/discovery/keys"
 TOKEN_SECRET="ZNh2N9toMwUVQxTVEe2ZnnMtgkh3xqKZ"
 ```
 
-> _**Note:**_ Do not use special characters for the GENERIC_CLIENT_ID as AD FS will split strings here, finally resulting in an identifier mismatch.
+{{% note %}}
+Do not use special characters for the `GENERIC_CLIENT_ID` as AD FS will split strings here, finally resulting in an identifier mismatch.
```

Comment:
Shall we spell out Active Directory Federation Services?

will split > splits

splits a string at the special character?

does omitting "finally" change the meaning?

Reply:
We define this above in the same document.